fluent-plugin-kubernetes-sumologic/README.md

```diff
@@ -25,33 +25,61 @@ can be used on pods:
-| tracing_container_name       | string | "container_name"                   | Traces       |                                                                                                                                                                                                                  |
-| tracing_format               | bool   | false                              | Traces       |                                                                                                                                                                                                                  |
-| tracing_host                 | string | "hostname"                         | Traces       |                                                                                                                                                                                                                  |
-| tracing_label_prefix         | string | "pod_label_"                       | Traces       |                                                                                                                                                                                                                  |
-| tracing_namespace            | string | "namespace"                        | Traces       |                                                                                                                                                                                                                  |
-| tracing_pod                  | string | "pod"                              | Traces       |                                                                                                                                                                                                                  |
-| tracing_pod_id               | string | "pod_id"                           | Traces       |                                                                                                                                                                                                                  |
+| Parameter Name                    | Type     | Default                            | Scope        | Description                                                                                                                                                                                                      |
+| --------------------------------- | -------- | ---------------------------------- | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Select the `Hide whitespace changes` checkbox in GitHub UI to see only relevant changes in the parameters table in README.